### PR TITLE
[Project Submission] Filecoin Flavored Ganache

### PR DESCRIPTION
**Pull request by Lizdaldal**

### CODE REVIEW

File: projects/developer_tooling/developer_tooling.md [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
 
 ## How the community can engage
-GitHub Discussion: <!--Start a disucssion with the community here: https://github.com/filecoin-project/community/discussions/new and attach the link!-->  
-Email:  
-Slack:  
-Twitter:  
-Discord:  
-Telegram:  
-WeChat:  
+Email: inquiry@trufflesuite.com
```
Comment (jennijuju) [inline projects/developer_tooling/developer_tooling.md:42]: Please create a project discussion, with the category as Show and tell - Developers and title as [Category] Project Name(i.e: [Application] Slate), link the discussion in the How the community can engage section. This is to create a place to interact with the Filecoin community, share your updates and gain feedback on your projects!

**Review by jennijuju — CHANGES_REQUESTED** [review]

**Comment by jennijuju** [comment]
Do you want your updates to be featured in Filecoin newsletter?

File: projects/developer_tooling/developer_tooling.md (hunk 1 above)
Comment (Lizdaldal) [reply projects/developer_tooling/developer_tooling.md:42]: Updated! Also opted into the newsletter

**Comment by Lizdaldal** [comment]
Updated - thanks!

**Review by jennijuju — APPROVED** [review]

**Pull request merged**